test/test_sam_annotator/test_annotator_2d.py

```diff
@@ -0,0 +1,54 @@
+import platform
+
+import numpy as np
+import pytest
+from skimage.data import binary_blobs
+
+import micro_sam.util as util
+from micro_sam.sam_annotator import annotator_2d
+
+
+def _check_layer_initialization(viewer, expected_shape):
```

Comment:
Could this function be refactored? It appears in every `test_...` file included with this PR. If they are all almost identical, we could make a util file for the tests. 

If the expected layer names change slightly (does that happen for the tracking annotator? idk, maybe) then that could be passed in as a kwarg to the function.

Reply:
Yes, good point. I will refactor it. (And I decided to keep the same layernames for the tracking annotator; that requires less changes in the code and I also think it's easier from a user perspective.)